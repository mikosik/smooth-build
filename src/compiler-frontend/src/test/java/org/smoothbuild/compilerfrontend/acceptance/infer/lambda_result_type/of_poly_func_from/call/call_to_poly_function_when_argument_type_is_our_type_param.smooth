A myIdentity(A a) = a;
myValue = (B b) -> myIdentity(b);