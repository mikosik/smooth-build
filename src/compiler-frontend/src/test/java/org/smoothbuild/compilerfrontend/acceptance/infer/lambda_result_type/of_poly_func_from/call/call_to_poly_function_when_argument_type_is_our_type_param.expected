A myIdentity<A>(A a)
  = a<>;
(A)->A myValue<A>
  = <B>(B b) -> myIdentity<B>(b<>)<A>;
